modules/tensor_mechanics/src/materials/RadialReturnStressUpdate.C

```diff
@@ -247,9 +257,6 @@ RadialReturnStressUpdateTempl<is_ad>::updateState(
                                                   ? std::sqrt(3.0 / 2.0 * dev_trial_stress_squared)
                                                   : 0.0;
 
-  // Set the value of 3 * shear modulus for use as a reference residual value
-  _three_shear_modulus = 3.0 * ElasticityTensorTools::getIsotropicShearModulus(elasticity_tensor);
```

Comment:
Maybe we should have something like this here if we're going to rely on the user calling `computeStressInitialize()` to set this:
```
mooseAssert(_three_shear_modulus != 0.0, "Shear modulus is zero. Ensure that computeStressInitialize() is called.");
```